inference-engine/tests/functional/plugin/cpu/single_layer_tests/convolution_backprop_data.cpp

```diff
@@ -258,132 +504,168 @@ const auto convParams_ExplicitPadding_Blocked_2D = ::testing::Combine(
-            ::testing::Values(Layout::ANY),
-            ::testing::Values(Layout::ANY),
-            ::testing::Values(std::vector<size_t >({ 2, 67, 7, 7 })),
-            ::testing::ValuesIn(emptyOutputShape),
-            ::testing::Values(CommonTestUtils::DEVICE_CPU)),
-        ::testing::ValuesIn(filterCPUInfoForDevice({conv_avx512_2D_1x1})),
+        convParams_ExplicitPadding_1x1_2D,
+        ::testing::ValuesIn(Planar_2D_inputs_smoke),
+        ::testing::Values(ElementType::f32),
         ::testing::ValuesIn(fusingParamsSet),
         ::testing::Values(cpuEmptyPluginConfig)),
     DeconvolutionLayerCPUTest::getTestCaseName);
 
 INSTANTIATE_TEST_SUITE_P(smoke_Deconv_2D_1x1_BF16, DeconvolutionLayerCPUTest,
     ::testing::Combine(
-        ::testing::Combine(
-            convParams_ExplicitPadding_1x1_2D,
-            ::testing::Values(Precision::FP32),
-            ::testing::Values(Precision::BF16),
-            ::testing::Values(Precision::BF16),
-            ::testing::Values(Layout::ANY),
-            ::testing::Values(Layout::ANY),
-            ::testing::Values(std::vector<size_t >({ 2, 67, 7, 7 })),
-            ::testing::ValuesIn(emptyOutputShape),
-            ::testing::Values(CommonTestUtils::DEVICE_CPU)),
-        ::testing::ValuesIn(filterCPUInfoForDevice({conv_avx512_2D_1x1})),
+        convParams_ExplicitPadding_1x1_2D,
+        ::testing::ValuesIn(Planar_2D_inputs_smoke),
+        ::testing::Values(ElementType::f32),
         ::testing::ValuesIn(fusingParamsSet),
         ::testing::Values(cpuBF16PluginConfig)),
     DeconvolutionLayerCPUTest::getTestCaseName);
 
-/* ========= */
+/* ============= Reorder + Deconvolution ============= */
+INSTANTIATE_TEST_SUITE_P(smoke_reorder_Deconv_2D, DeconvolutionLayerCPUTest,
+    ::testing::Combine(
+        ::testing::Combine(::testing::ValuesIn(kernels2d),
+                           ::testing::Values(InferenceEngine::SizeVector{1, 1}),
+                           ::testing::ValuesIn(padBegins2d),
+                           ::testing::ValuesIn(padEnds2d),
+                           ::testing::ValuesIn(dilations2d),
+                           ::testing::ValuesIn(numOutChannels_Blocked),
+                           ::testing::Values(ngraph::op::PadType::EXPLICIT),
+                           ::testing::ValuesIn(emptyOutputPadding)),
+        ::testing::Values(DeconvInputData{InputShape{{-1, 67, -1, -1}, {{ 2, 67, 7, 7}, { 2, 67, 5, 7}, { 1, 67, 9, 4}}},
+                                                     ngraph::helpers::InputLayerType::PARAMETER,
+                                                     {{15, 15}, {9, 10}, {9, 9}},
+                                                     filterCPUInfoForDevice({conv_avx512_2D})}),
+        ::testing::Values(ElementType::f32),
+        ::testing::Values(emptyFusingSpec),
+        ::testing::Values(cpuEmptyPluginConfig)),
+    DeconvolutionLayerCPUTest::getTestCaseName);
 
 } // namespace
+
```

Comment:
I could not find any test case that tests auto padding handling in the dynamic shapes case. Could you please add such test cases here and for the GroupDecovolution? You can use CPU specific SL tests for the Convolution as a reference.